bluesky_queueserver/manager/comms.py

```diff
@@ -779,6 +859,11 @@ class ZMQCommSendAsync:
         specifying ``raise_exceptions`` parameter of ``send_message()`` function. The exception
         ``CommTimeoutError`` is raised if the parameter is ``True``, otherwise error message
         is returned by ``send_message()``.
+    server_public_key : str or None
+        Server public key (z85-encoded 40 character string). Valid public key from server
```

Comment:
This line may need to be sync'd with the one we discussed earlier.
```suggestion
        Server public key (z85-encoded 40 character string). The valid public key from the server
```